admin_panel/src/commonMain/kotlin/net/thechance/mena/admin_panel/presentation/component/navigation_bar/VerticalNavigationBar.kt

```diff
@@ -0,0 +1,36 @@
+package net.thechance.mena.admin_panel.presentation.component.navigation_bar
+
+import androidx.compose.foundation.background
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableIntStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Modifier
+import net.thechance.mena.designsystem.presentation.theme.theme.Theme
+
+@Composable
+fun VerticalNavigationBar(
+    modifier: Modifier = Modifier,
+    initialSelectedIndex: Int = 0,
+    content: @Composable VerticalNavigationScope.() -> Unit = {},
+) {
+    val scope = remember { VerticalNavigationScopeImpl() }.apply {
+        items.clear()
+        content()
+    }
+
+    var selectedItemIndex by remember {
+        mutableIntStateOf(initialSelectedIndex)
+    }
```

Comment:
format this in one line
```suggestion
    var selectedItemIndex by remember { mutableIntStateOf(initialSelectedIndex) }
```

Reply:
i cant its too long
<img width="1022" height="57" alt="image" src="https://github.com/user-attachments/assets/9caa130d-61e7-43f0-82c6-9f56287d9955" />